modules/task/README.md

```diff
@@ -0,0 +1,49 @@
+# Async task module
+
+The async task module presents an distributed task execution framework
+for services that want to execute a function asynchronously and in a durable
+fashion. It achieves durability with "backends" that are messaging transports.
```

Comment:
It's tied to the code. Also though technically it should be back ends, I've observed it being used as one word very frequently in other documentation.